pkg/morserino/channels.go

```diff
@@ -22,30 +22,25 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 */
 
-import (
-	"fmt"
-	"strings"
-)
+// Structure containing all the channels used by the application
+type MorserinoChannels struct {
+	// Channel for the data flow from serial port to display goroutines
+	MessageBuffer chan string
 
-//FIXME: Add comment
-type ConsoleDisplay struct {
-	currentLine string
-	newLine     string
-}
+	// Channel indicating that all data has been displayed and that the
+	// application can be closed
+	DisplayCompleted chan bool
```

Comment:
```suggestion
	DisplayCompleted chan struct{}
```

If you don't care about the actual data carried inside the channel, use `chan struct{}` instead (literally chan of empty struct} which doesn't allocate any space for the data.

You can use it like this

```golang
c := make(chan struct{})
c <- struct{}{}
```